config/locales/views/annotations/en.yml

```diff
@@ -1,8 +1,12 @@
 en:
   annotations:
+    count: 'Number of times used: '
     help_message: "Click on an annotation category to add annotations."
     last_edited_by: "Last Edited By:"
     text: "Text"
+    usage: 'Usage Details'
+    used_by: 'Used By'
+    used_times: "Used %{number} times"
```

Comment:
Strings with counts are always a bit awkward (think "Used 1 times"). The standard way to resolve this is to have nested keys under `used_times`. Search "pluralization" at https://github.com/fnando/i18n-js.